packages/sdk/identity/src/odis/identifier.ts

```diff
@@ -45,7 +45,7 @@ export enum IdentifierPrefix {
   NULL = '',
   PHONE_NUMBER = 'tel',
   EMAIL = 'mailto',
-  TWITTER = 'twit',
```

Comment:
I dont think these can be changed.

@alecps

Reply:
+1 would make any identifiers created with the previous prefix not interoperable with a new one, also I believe `twit` is used since it's a specified DID method